go/parquet/go.sum

```diff
@@ -0,0 +1,147 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
what is this file?

Reply:
go.sum is similar to a yarn.lock or Pipfile.lock if you're familiar with node/python development. It's a list of checksums of the dependencies that are specified in the go.mod.